app/controllers/games_controller.rb

```diff
@@ -4,6 +4,16 @@ def index
   end
 
   def show
-    # code goes here
+    # code to get the board on my page. 24 is the game id that had all my pieces so I hardcoded it
+    @game = Game.find_by(id: 29)
```

Comment:
Before this gets merged, can you change this to `@game = Game.find(params[:id])`?  For testing then you could always visit `/games/29` to grab your particular test game.